ui/packages/consul-ui/app/models/auth-method.js

```diff
@@ -0,0 +1,24 @@
+import Model, { attr } from '@ember-data/model';
+
+export const PRIMARY_KEY = 'uid';
+export const SLUG_KEY = 'Name';
+
+export default class AuthMethod extends Model {
+  @attr('string') uid;
+  @attr('string') Name;
+
+  @attr('string') Datacenter;
+  @attr('string') Namespace;
+  @attr('string', { defaultValue: () => '' }) Description;
+  @attr('string', { defaultValue: () => '' }) DisplayName;
+  @attr('string', { defaultValue: () => '' }) TokenLocality;
+  @attr('string') Type;
+  @attr('string') Host;
+  @attr('string') ServiceAccountJWT;
+  @attr('string') CACert;
+  @attr('string') MaxTokenTTL;
+  @attr('number') CreateIndex;
+  @attr('number') ModifyIndex;
+  @attr() Datacenters; // string[]
```

Comment:
From reading the API docs I'm not massively sure if some of the things here are part of a `Config` object? (So `Host`,`ServiceAccountJWT`, `CACert`).

Playing with it a little it also looks as though the `MaxTokenTTL` isn't available on the listing endpoint - which means we won't be able to show that in the listings page. Probably a good idea to check that with the rest of the team.